activities.json

```diff
@@ -306,6 +306,19 @@
     "title": "CSS Properties and Values API: @property",
     "url": "https://drafts.css-houdini.org/css-properties-values-api-1#at-property-rule"
   },
+  {
+    "ciuName": "css-has-selector",
+    "description": ":has selector lets authors select elements that anchor at least one element that matches its inner relative selector.",
+    "id": "css-has-selector",
+    "mdnUrl": "https://developer.mozilla.org/en-US/docs/Web/CSS/:has",
+    "mozBugUrl": "https://bugzilla.mozilla.org/show_bug.cgi?id=has-pseudo",
+    "mozPosition": "positive",
+    "mozPositionDetail": "We acknowledge the developer demand for this selector, and the implementation is shipping as of 121. However, we are still concerned about the worst-case performance of DOM tree mutation when using the selector.",
```

Comment:
I think we generally try to not reference Firefox shipping status in standards-positions rationale. Try to answer if the feature is good for the web.

Also, I think this comes across as basically negative. "We had to implement it, but we don't like it."

Reply:
How about:

```suggestion
    "mozPositionDetail": "We recognize that :has is something that a lot of people want. That power does come with performance costs, like the potential the worst-case performance resulting from DOM tree mutation to be severe. Overall, the utility of the selector justifies this risk, but we might need to do more to help developers avoid the worst problems.",
```